src/xtopcom/xblockmaker/src/xproposal_maker.cpp

```diff
@@ -134,6 +134,9 @@ int xproposal_maker_t::verify_proposal(base::xvblock_t * proposal_block, base::x
         xwarn("xproposal_maker_t::verify_proposal fail clone tablestate. %s,cert_height=%" PRIu64 "", cs_para.dump().c_str(), proposal_prev_block->get_height());
         return xblockmaker_error_proposal_table_state_clone;
     }
+
+    get_txpool()->update_receiptid_state(proposal_block->get_account(), tablestate->get_receiptid_state());
```

Comment:
txpool should update committed state, here is highqc table state

Reply:
fixed